nc-news/src/App.js

```diff
@@ -19,23 +19,29 @@ function App() {
     avatar_url:
       "https://vignette.wikia.nocookie.net/mrmen/images/7/78/Mr-Grumpy-3A.PNG/revision/latest?cb=20170707233013",
   });
+
+  const [topics, setTopics] = useState([]);
+
   return (
     <BrowserRouter>
       <UserContext.Provider value={{ user, setUser }}>
-        <div className="App">
-          <Navbar />
-          <Routes>
-            <Route path="/" element={<Home />}></Route>
-            <Route path="/topics" element={<TopicsList />}></Route>
-            <Route path="/users" element={<ArticlesListByUser />}></Route>
-            <Route path="/users/user_change" element={<UserChange />}></Route>
-            <Route path="/articles" element={<ArticleList />}></Route>
-            <Route
-              path="/api/articles/:article_id"
-              element={<ArticleCard />}
-            ></Route>
-          </Routes>
-        </div>
+        <TopicsContext.Provider value={{ topics, setTopics }}>
+          <div className="App">
+            <Navbar />
+            <Routes>
+              <Route path="/" element={<Home />}></Route>
+              <Route path="/topics" element={<TopicsList />}></Route>
+              <Route path="/topics/:topic" element={<ArticleList />}></Route>
+              <Route path="/users" element={<ArticlesListByUser />}></Route>
+              <Route path="/users/user_change" element={<UserChange />}></Route>
+              <Route path="/articles" element={<ArticleList />}></Route>
+              <Route
+                path="/api/articles/:article_id"
```

Comment:
Remember this is a FE URL and doesn't need to be the same as BE, this path should be /articles/:article_id